changelog.d/lang-server.added

```diff
@@ -0,0 +1 @@
+Semgrep LS will now not show findings that have been ignored in CI
```

Comment:
I'm not sure user are familiar with what LS mean. Maybe say Semgrep language server, or
Semgrep LSP (I know we should use LS, because LSP is the protocol, but I think most people
are familiar with what LSP mean but not LS).

Reply:
What means CI here? You mean ignored in the Semgrep App?

Reply:
Yup, meant the app, will update this accordingly